docs/book/src/reference/envtest.md

```diff
@@ -97,12 +97,12 @@ You can use environment variables and/or flags to specify the `kubectl`,`api-ser
 ### Environment Variables
 
 | Variable name | Type | When to use |
-| --- | :--- | :--- |
-| `USE_EXISTING_CLUSTER` | boolean | Instead of setting up a local control plane, point to the control plane of an existing cluster. |
-| `KUBEBUILDER_ASSETS` | path to directory | Point integration tests to a directory containing all binaries (api-server, etcd and kubectl). |
-| `TEST_ASSET_KUBE_APISERVER`, `TEST_ASSET_ETCD`, `TEST_ASSET_KUBECTL` | paths to, respectively, api-server, etcd and kubectl binaries | Similar to `KUBEBUILDER_ASSETS`, but more granular. Point integration tests to use binaries other than the default ones. These environment variables can also be used to ensure specific tests run with expected versions of these binaries. |
-| `KUBEBUILDER_CONTROLPLANE_START_TIMEOUT` and `KUBEBUILDER_CONTROLPLANE_STOP_TIMEOUT` | durations in format supported by [`time.ParseDuration`](https://golang.org/pkg/time/#ParseDuration) | Specify timeouts different from the default for the test control plane to (respectively) start and stop; any test run that exceeds them will fail. |
-| `KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT` | boolean | Set to `true` to attach the control plane's stdout and stderr to os.Stdout and os.Stderr. This can be useful when debugging test failures, as output will include output from the control plane. |
+| --- | :--- | :---                                                                                                                                                                                                                                                    |
+| `USE_EXISTING_CLUSTER` | boolean | Instead of setting up a local control plane, point to the control plane of an existing cluster.                                                                                                                                                         |
```

Comment:
```suggestion                                                                                                                                       |
```
I did not check in the C+R code but I am not sure if we have this env var config implemented there. How to use this option is described in the new section. Anyway, I do not think that we should add this one here since it can brings confusion because the section has the purpose to explain its config setup.